lab12_python.py

```diff
@@ -0,0 +1,65 @@
+import os
+import hashlib
+import threading
+
+def duplicates(folder):
```

Comment:
using hash only will not do the job, the comparison will take a lot of time anyway